resources/qml/UserSettingsPage.qml

```diff
@@ -0,0 +1,218 @@
+// SPDX-FileCopyrightText: 2021 Nheko Contributors
+// SPDX-FileCopyrightText: 2022 Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import "ui"
+import Qt.labs.platform 1.1 as Platform
+import QtQuick 2.15
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+import QtQuick.Window 2.13
+import im.nheko 1.0
+
+Rectangle {
+    id: userSettingsDialog
+
+    property bool collapsed: width < 800
+    color: Nheko.colors.window
+
+    Shortcut {
+        sequence: StandardKey.Cancel
+        onActivated: userSettingsDialog.close()
+    }
+
+    ScrollView {
+        id: scroll
+
+        palette: Nheko.colors
+        ScrollBar.horizontal.visible: false
+        anchors.fill: parent
+        anchors.margins: Nheko.paddingLarge
+
+        contentWidth: availableWidth
+
+        Timer {
+            id: deadTimer
+            interval: 500
+        }
+
+        Connections {
+            target: scroll.contentItem
+            function onContentYChanged() { deadTimer.restart(); }
+        }
+
+
+        GridLayout {
+            id: grid
+
+            columns: userSettingsDialog.collapsed ? 1 : 2
+            rowSpacing: Nheko.paddingMedium
+            columnSpacing: Nheko.paddingMedium
+
+            anchors.fill: parent
+            anchors.leftMargin: userSettingsDialog.collapsed ? Nheko.paddingLarge : (userSettingsDialog.width-600) * 0.4
```

Comment:
This uses a width of 600 while the collapsed logic uses a width of 800.  What's the rationale for the difference?

Reply:
Copy and paste error, because that should have been a constant.... .-.